.github/workflows/lint.yml

```diff
@@ -44,14 +44,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Install rustc nightly
+      - name: Install rustc stable
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly
+          toolchain: stable
```

Comment:
Why we are switching to stable? It seems we are using nightly for docs, since docs.rs does the same. CI must check if the docs can be published to docs.rs.

Reply:
I aligned it with the one in rust-amplify. Now I switched back to nightly

Reply:
Seems like I need to fix that in amplify